conference.js

```diff
@@ -71,7 +72,7 @@ const eventEmitter = new EventEmitter();
 let room;
 let connection;
 let localAudio, localVideo;
-let initialAudioMutedState = false, initialVideoMutedState = false;
+let initialAudioMutedState = false;
```

Comment:
I finally got around to testing just this specific change. I recall initialVideoMutedState was added to fix a race condition with the iframe api not honoring video muting (commit ab6269). On your branch when I initialize the api and immediately call api.executeCommand('toggleVideo') my user is not video muted. Is it okay to let muting not happen?